src/Modules/controllTools.js

```diff
@@ -0,0 +1,46 @@
+/* eslint-disable import/no-cycle */
+/* eslint-disable no-restricted-globals */
+/* eslint-disable import/prefer-default-export */
+
+import { getTask } from './data.js';
+
+export function deleteTask() {
+  const clearTask = document.querySelector('.clear-task');
+  clearTask.addEventListener('click', () => {
+    if (getTask().length > 0) {
+      const filterCompliteTask = getTask().filter(
+        (task) => task.completed !== true,
+      );
+      localStorage.setItem('Task-list', JSON.stringify(filterCompliteTask));
+      location.reload();
+    }
+  });
+}
+
+export function addTask() {
+  class Task {
+    constructor(description) {
+      this.description = description;
+      this.index = new Date();
```

Comment:
- [ ] **Required:** The obj `index` should not be a date object but a number. That can be updated.

- [ ] **Required:** The index should update when an item is deleted so that if there are 4 list item, the index should read 1, 2, 3 and 4

![image](https://user-images.githubusercontent.com/27709832/154710811-1bc93e84-ff76-4b84-bbdc-43747f848556.png)